<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumidityWindow</class>
 <widget class="QMainWindow" name="HumidityWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>27</y>
      <width>90</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/temperature_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/humidity_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QLCDNumber" name="temp_lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>200</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>10</y>
       <width>20</width>
       <height>120</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Aharoni</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Â°</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="temp_lcdNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>200</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="temp_lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>270</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>30</y>
       <width>50</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Aharoni</family>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="humi_lcdNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>200</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="humi_lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>270</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>20</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>791</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>791</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="humi_Save"/>
    <addaction name="humi_Open"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="humi_SourceFile"/>
    <addaction name="humi_ShowInPlot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Data"/>
  </widget>
  <action name="humi_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="humi_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="humi_SourceFile">
   <property name="text">
    <string>SourceF&amp;ile</string>
   </property>
  </action>
  <action name="humi_ShowInPlot">
   <property name="text">
    <string>ShowIn&amp;Plot</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
